.opencode/opencode.jsonc

```
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://opencode.ai/config.json",
+  "permission": {
```

Biome (2.4.12)

[error] The key permission was already declared.

(lint/suspicious/noDuplicateObjectKeys)